### PR TITLE
gpexpand: remove explicit set of client encoding to utf-8

### PR DESCRIPTION
**Pull request by nadeemg**
gpexpand was the only utility explicitly setting encoding to utf-8, and failing on a database with WIN874 encoding containing data in Thai chars. This change removes the explicit set.

### CODE REVIEW

**Comment by nadeemg** [comment]
> Do we have a good idea of why UTF-8 encoding was being set explicitly?

No, we don't.

File: gpMgmt/test/behave_utils/utils.py [hunk 1]
```diff
@@ -255,25 +257,39 @@ def check_db_exists(dbname, host=None, port=0, user=None):
...
 
     return False
 
 
-def create_database_if_not_exists(context, dbname, host=None, port=0, user=None, saveConnInfoInContext=False):
-    if not check_db_exists(dbname, host, port, user):
-        create_database(context, dbname, host, port, user)
+def create_database_if_not_exists(context, dbname, host=None, port=0, user=None, saveConnInfoInContext=False, encoding=None):
+    if not check_db_exists(dbname, host, port, user, encoding):
+        create_database(context, dbname, host, port, user, encoding)
```
[the remark this answers is not in the document]
Comment (nadeemg) [reply gpMgmt/test/behave_utils/utils.py]: Yeah, which is why the test is explicitly dropping the database before trying to create it. 

Would it be better if the check_db_exists func ignore encoding?

**Comment by chrishajas** [comment]
It appears that this was trying to fix some bug in the historical repo (there’s an associated jira), possibly 

"table name has non ascii/unicode characters with table names having line separators '\u2028’ "

File: gpMgmt/test/behave_utils/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jchampio) [reply gpMgmt/test/behave_utils/utils.py]: > Yeah, which is why the test is explicitly dropping the database before trying to create it.

Ah, gotcha. Is the error message that we get back in this case helpful, or is it just "database already exists"?

> Would it be better if the check_db_exists func ignore encoding?

Only if something else explicitly checks that the encoding is what the developer wanted it to be, I think.

**Comment by nadeemg** [comment]
> It appears that this was trying to fix some bug in the historical repo (there’s an associated jira), possibly
> 
> "table name has non ascii/unicode characters with table names having line separators '\u2028’ "

Thanks for the pointer @chrishajas. That can't find that issue in JIRA, but I'll take another look.

**Pull request closed without merging**